.circleci/config.yml

```diff
@@ -70,19 +70,19 @@ jobs:
     <<: *defaults
     working_directory: /home/fossa/go/src/github.com/fossas/fossa-cli
     docker:
-      - image: quay.io/fossa/fossa-cli-test:circleci
+      - image: quay.io/fossa/fossa-cli-test-base:circleci
     steps:
+      - checkout
+      - run: 
+          name: Compile fossa binary
+          command: make
       - run:
           name: Make folders
           command: |
             mkdir -p $TEST_RESULTS
             mkdir -p $ARTIFACTS
       - attach_workspace:
```

Comment:
You'll also want to remove this step, and the steps in the `build` stage for saving to the workspace. It'll help to go through the whole configuration and maybe read the Circle CI documentation on workspaces and steps.